src/scan/postgres_scan.cpp

```diff
@@ -157,12 +182,12 @@ PostgresScanTableFunction::PostgresScanTableFunction()
 	to_string = ToString;
 }
 
-std::string
-PostgresScanTableFunction::ToString(const duckdb::FunctionData *data) {
-	auto &bind_data = data->Cast<PostgresScanFunctionData>();
-	std::ostringstream oss;
-	oss << "(POSTGRES_SCAN) " << GetRelationName(bind_data.rel);
-	return oss.str();
+duckdb::InsertionOrderPreservingMap<duckdb::string>
+PostgresScanTableFunction::ToString(duckdb::TableFunctionToStringInput &input) {
+	auto &bind_data = input.bind_data->Cast<PostgresScanFunctionData>();
+	duckdb::InsertionOrderPreservingMap<duckdb::string> result;
+	result["Table"] = GetRelationName(bind_data.rel);
```

Comment:
Maybe interesting to note that it is a PG table?
```suggestion
	result["Table"] = GetRelationName(bind_data.rel) + " (PG)";
```

Reply:
Explain already show it is `POSTGRES_SCAN` node. We can add it if needed.